test/images/scripts/debian-stable.bootstrap

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh -ex
+ARCH=x86_64
+DEBIAN_LATEST=$(virt-builder -l  | grep "$ARCH" | sort -r | grep -m1 '^debian-' | cut -d' ' -f1)
+exec $(dirname $0)/lib/debian.bootstrap "$1" "$2" "$DEBIAN_LATEST" "deb http://deb.debian.org/debian stable main"
```

Comment:
Ahh, $DEBIAN_LATEST is "debian-8", can that be used in sources.list instead of the code name?